fitbenchmarking/fitbenchmarking
# Adding sanitised name attribute to FittingProblem

#### Description of Work

Fixes #397 

Introduces a `sanitised_name` attribute to fitting problem.

#### Testing Instructions

1. Check plots links work.
2. `sanitised_name` is used where necessary. 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
